packages/web-components/src/toggle-button/toggle-button.stories.ts

```diff
@@ -1,128 +1,210 @@
     },
     pressed: {
       control: 'boolean',
-      table: {
-        type: {
-          summary: 'Sets the pressed state of the component',
-        },
-        defaultValue: {
-          summary: 'false',
-        },
-      },
+      description: 'Sets the pressed state of the component',
+      name: 'pressed',
+      table: { category: 'attributes', type: { summary: 'boolean' } },
     },
-    disabled: {
-      control: 'boolean',
+    size: {
+      control: 'select',
+      description: 'The size of the button.',
+      options: ['', ...Object.values(ToggleButtonSize)],
+      mapping: { '': null, ...ToggleButtonSize },
       table: {
-        type: {
-          summary: 'Sets the disabled state of the component',
-        },
-        defaultValue: {
-          summary: 'false',
-        },
+        category: 'attributes',
+        type: { summary: Object.values(ToggleButtonSize).join('|') },
       },
     },
-    disabledFocusable: {
-      control: 'boolean',
+    shape: {
+      control: 'select',
+      description: 'The shape of the button.',
+      options: ['', ...Object.values(ToggleButtonShape)],
+      mapping: { '': null, ...ToggleButtonShape },
       table: {
-        type: {
-          summary: 'The component is disabled but still focusable',
-        },
-        defaultValue: {
-          summary: 'false',
-        },
+        category: 'attributes',
+        type: { summary: Object.values(ToggleButtonShape).join('|') },
       },
     },
-    content: {
-      control: 'Button text',
+    value: {
+      control: 'text',
+      description: 'The initial value of the button.',
+      table: { category: 'attributes', type: { summary: 'string' } },
+    },
+    slottedContent: {
+      control: false,
+      description: 'The default slot',
+      name: '',
+      table: { category: 'slots', type: {} },
+    },
+    startSlottedContent: {
+      control: false,
+      description: 'Slot for start icons',
+      name: 'start',
+      table: { category: 'slots', type: {} },
+    },
+    endSlottedContent: {
+      control: false,
+      description: 'Slot for end icons',
+      name: 'end',
+      table: { category: 'slots', type: {} },
     },
   },
-} as ToggleButtonStoryMeta;
+} as Meta<FluentToggleButton>;
 
-export const Button = renderComponent(storyTemplate).bind({});
+export const Default: Story = {};
 
-export const Appearance = renderComponent(html<ToggleButtonStoryArgs>`
-  <fluent-toggle-button>Default</fluent-toggle-button>
-  <fluent-toggle-button appearance="primary">Primary</fluent-toggle-button>
-  <fluent-toggle-button appearance="outline">Outline</fluent-toggle-button>
-  <fluent-toggle-button appearance="subtle">Subtle</fluent-toggle-button>
-  <fluent-toggle-button appearance="transparent">Transparent</fluent-toggle-button>
-`);
+export const Appearance: Story = {
+  render: renderComponent(html`<div>${repeat(story => story.storyItems, storyTemplate)}</div> `),
```

Comment:
nit: the wrapping `<div>`s probably aren't necessary

Reply:
In pre storybook 8 it is, othwerwise the code preview does not work.